plugin.rb

```diff
@@ -133,7 +133,15 @@ def self.accept_answer!(post, acting_user, topic: nil)
           )
         end
 
-        auto_close_hours = SiteSetting.solved_topics_auto_close_hours
+        if topic&.category.present?
+          auto_close_hours =
+            topic&.category&.custom_fields&.[]("solved_topics_auto_close_hours").to_i
```

Comment:
Could we use this syntax instead? `topic&.category&.custom_fields&.["solved_topics_auto_close_hours"].to_i` Looks like we are using this style in a few other places in this file.

Reply:
Oh okay! Sounds good. I guess there is some magic happening with the .[]() syntax.